oktoberfest/runner.py

```diff
@@ -75,6 +86,12 @@ def _preprocess(spectra_files: List[Path], config: Config) -> List[Path]:
         else:
             internal_search_file = config.search_results
             search_results = pp.load_search(internal_search_file)
+            # convert custom modifications to internal
+
+            if custom_static_mods is not None:
```

Comment:
same a above, the function pp.load_search should perform everything that is necessary, i.e. allow it to accept the config.var_mods cand config.static_mods and don't postprocess the results here, as it defies the concept of having standlone API functions that do everything the user needs without having to manually modify the result.